modules/configuring-a-proxy-after-installation-ocm.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-a-proxy-after-installation-ocm_{context}"]
+= Configuring a proxy after installation using {cluster-manager}
+
+You can use {cluster-manager-first} to add a cluster-wide proxy configuration to an existing {product-title} cluster in a Virtual Private Cloud (VPC). You can enable a proxy only for clusters that are deployed by using the Customer Cloud Subscription (CCS) model.
+
+You can also use {cluster-manager} to update an existing cluster-wide proxy configuration. For example, you might need to update the network address for the proxy or replace the additional trust bundle if any of the certificate authorities for the proxy expire.
+
+[IMPORTANT]
+====
+The cluster applies the proxy configuration to the control plane and compute nodes. While applying the configuration, each cluster node is temporarily placed in an unschedulable state and drained of its workloads. Each node is restarted as part of the process.
+====
+
+.Prerequisites
+
+* You have an {product-title} cluster that uses the Customer Cloud Subscription (CCS) model.
+* Your cluster is deployed in a VPC.
+
+.Procedure
+
+. Navigate to {cluster-manager-url} and select your cluster.
+
+. Under the *Virtual Private Cloud (VPC)* section on the *Networking* page, click *Edit cluster-wide proxy*.
+
+. On the *Edit cluster-wide proxy* page, provide your proxy configuration details:
+.. Complete at least one of the following fields:
+** Specify a valid *HTTP proxy URL*.
+** Specify a valid *HTTPS proxy URL*.
+** Optional: In the *No proxy domains* field, specify any domains or Classless Inter-Domain Routing (CIDR) ranges that should not use the proxy. To exclude all subdomains for a domain, you can use `\*` as a wildcard. For example, you can specify `*.example.com` to to prevent all traffic for the example.com domain and its subdomains from using the proxy.
```

Comment:
Currently,the no_proxy is not supported on UI page.I believe we should expose this after it is allowed on UI

Reply:
This docs PR is dependent on https://issues.redhat.com/browse/HAC-698 and https://issues.redhat.com/browse/HAC-710 and will be merged after the feature is promoted.

Reply:
@yingzhanredhat hi! I have now tested the **Networking** tab proxy updates in stage, following the recent merge. I have subsequently removed the **No proxy domains** steps as per your suggestion.